src/components/Body.js

```diff
@@ -5,9 +5,13 @@ import PropTypes from "prop-types";
 
 export default class Body extends React.Component {
   render() {
-    const markSchemeArray = this.props.markSchemeElements.map((element, i) => (
-      <li key={i}> {element.text} </li>
-    ));
+    const markSchemeArray = this.props.markSchemeElements.map(
+      (element, index) => (
+        <li key={index} onClick={() => this.props.markComplete(index)}>
+          {element.text} <p>{element.completed.toString()}</p>
```

Comment:
Perhaps put a comment making clear the `<p>` tag is temporary and will probably be replaced with a `className` that changes based on the `markComplete` value.

Reply:
done